libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_subscription.c

```diff
@@ -261,6 +270,80 @@ static void _blockingCallback( void * pArgument,
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Setting the MQTT Context for the given MQTT Connection.
+ *
+ */
+static void _setContext( IotMqttConnection_t pMqttConnection )
```

Comment:
Is there a way this function can be shared across all the tests?

Reply:
Discussed